src/host/_stream.cpp

```diff
@@ -128,15 +105,11 @@ static void AdjustCursorPosition(SCREEN_INFORMATION& screenInfo, _In_ til::point
         LOG_IF_FAILED(screenInfo.SetViewportOrigin(false, WindowOrigin, true));
     }
 
-    if (interactive)
-    {
-        screenInfo.MakeCursorVisible(coordCursor);

```

Comment:
this was removed, but was only ever used for 
* `COOKED_READ_DATA::_writeChars`
* `COOKED_READ_DATA::_handlePostCharInputLoop`